Vignettes/usingcchsflow.Rmd

```diff
@@ -0,0 +1,176 @@
+---
+title: "Transform CCHS variables"
+date: 'Last updated: 2019-08-27'
+output: rmarkdown::html_vignette
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Getting Started
+### Loading the necessary packages
+
+The `RecWTable` and `SetDataLabels` functions are part of the [bllflow](https://bllflow.projectbiglife.ca/index.html) package. `RecWTable` is used to transform CCHS variables to a common name, while `SetDataLabels` reapplies labels that are lost during the combination process after transforming each CCHS dataset. You must install & load the package onto your environment in order to do any transformations. To be able to combine datasets with minimal issues, we recommend using the `bind_rows` function in the [dplyr](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8) package. 
+
+```{r results = 'hide', message = FALSE, warning =FALSE}
+library(bllflow)
+library(dplyr)
+```
+
+For this vignette, we will use the `datatable` function in the [DT](https://rstudio.github.io/DT/) package to provide data tables to illustrate the transformations.
+```{r results = 'hide', message = FALSE, warning =FALSE}
+library(DT)
+```
+
+## Steps
+### Step 1. Loading the CCHS datasets, variables.csv, and variableDetails.csv
+
+You will need to load the CCHS datasets as well as the two variable worksheets before you are able to do any transformations. Since CCHS datasets cannot be shared publicly, we will create mock CCHS datasets for the years 2001 and 2013 which will be used in examples later on. 
+
+```{r}
+varSheet <- read.csv(file.path(getwd(), '../inst/extdata/variables.csv'))
+varDetails <- read.csv(file.path(getwd(), '../inst/extdata/variableDetails.csv'))
+cchsMock2001 <- data.frame(DHHA_SEX = c(2, 1, 1, 6), DHHAGAGE = c(3, 4, 6, 6), FVCADTOT = c(12, 4, 20, 9))
+cchsMock2013 <- data.frame(DHH_SEX = c(1, 2, 1), DHHGAGE = c(2, 1, 1), FVCDTOT = c(25, 15, 6))
+```
+
+### Step 2. Creating a transformed dataset for each CCHS cycle
+
+To call the `RecWTable` function to transform the variables of a CCHS dataset:
+
+1. Create a name for your transformed dataset (e.g. cchsYear1)
+2. Indicate the database of the CCHS dataset in the `dataSource` argument
+3. Indicate the name of `variableDetails.csv` in the `variableDetails` argument
```

Comment:
Change to `variableDetails.csv`

Reply:
isnt it already variableDetails.csv? Or do u mean the argument reference? If u mean the later that wouldnt make sense since variableDetails is RecWTable argument